amundsen_application/static/js/components/common/Card/styles.scss

```diff
@@ -4,7 +4,7 @@
 $shimmer-loader-items: 1, 2, 3, 4, 5;
 $shimmer-loader-row-height: 16px;
 $shimmer-loader-row-min-width: 90;
```

Comment:
You could remove this one too

Reply:
$shimmer-loader-row-min-width still used on line 79 for now, do you want to remove that too?